nemo/collections/nlp/data/language_modeling/text_memmap_dataset.py

```diff
@@ -31,11 +33,49 @@
 __idx_suffix__ = 'idx'  # index file suffix
 
 
+def _create_short_hash(string: str) -> str:
+    """Utility function to compute hash of a string.
+
+    This is used to create unique filenames from a full path.
+
+    Args:
+        string: the string to create a hash for.
+    Returns:
+        A short hash of the string.
+    """
+    # Compute the SHA-256 hash of the string
+    sha256_hash = hashlib.sha256(string.encode()).hexdigest()
+
+    # Take a portion of the hash to create a short hash
+    short_hash = sha256_hash[:8]
```

Comment:
Why use short hash? Why take the risk of collision?

Reply:
As discussed, I removed this part.